src/components/App/App.js

```diff
@@ -23,26 +25,63 @@ class App extends Component {
     })
   }
 
-  addPokemon = (queriedPokemon) => {
-    this.validatePokemonData(queriedPokemon)
-    //set state with the validated pokemon
-  }
-
-  validatePokemonData = (queriedPokemon) => {
-    const lowerCaseInput = queriedPokemon.queriedPokemon.toLowerCase()
-
+ addPokemon = (queriedPokemon) => {
+   this.validatePokemonData(queriedPokemon)
+   //set state with the validated pokemon
+ }
+ 
+  validatePokemonName = (queriedPokemon) => {
+    console.log('reaches name function') 
+    const lowerCaseInput = queriedPokemon.toLowerCase()
     const verifiedName = this.state.pokeDex.find(pokemon => {
       let lowerCaseName = pokemon.name.toLowerCase()
       if (lowerCaseName.includes(lowerCaseInput) && lowerCaseInput !== '')  {
         return pokemon
       }
     })
-
     if (verifiedName === undefined) {
-      console.log('No Good NAMe!')
+      console.log('No Good NAME!')
+    } else {
+      console.log(verifiedName.name)
+      return verifiedName.name
+    }
+  }
+
+  pokemonApiFetch(searchTerm) {
+    console.log('reaches api fetch')
+    getPokemonData(searchTerm)
+    .then(data => {
+      this.setState({
+        foundPokemon: data
+      })
+    })
+    console.log(this.state.foundPokemon)
+    return this.state.foundPokemon
+  }
+
+  validatePokemonID(queriedPokemon) {
+    console.log('reaches id function')
+    const parsedID = parseInt(queriedPokemon);
+
+    if (parsedID > 0 && parsedID < 152) {
+      // this.pokemonNameFetch(parsedID)
+      console.log(parsedID)
+      // console.log(this.state.foundPokemon)
+      return parsedID;
+
+    } else {
+      console.log('No Good ID!')
+      return 'No Good ID!'
+    }
+ }
+ 
+  validatePokemonData = (queriedPokemon) => {
+    if (!isNaN(queriedPokemon.queriedPokemon)) {
+      const idValidated = this.validatePokemonID(queriedPokemon.queriedPokemon)
+      this.pokemonApiFetch(idValidated)
```

Comment:
I love that you broke this up into more modular functions with single responsibilities.